content-services/content-sources-backend
# Fixes 1394: Add sentry support

## Summary
This adds sentry/glitchtip support for alerts on errors, panics, and fatal messages

## Testing steps
I tried setting up glitchtip locally, and fought my way through email setup issues, and ended up testing with glitchtip directly.  So you can login to glitchtip grab the DSN for hcc-content-sources-stage and run the server locally with  SENTRY_DSN="FROM_GLITCHTIP"  make run

Then trigger an error.  Everyone should get a notification, so only do this sparingly :)
